cli/installer/install-azd.ps1

```diff
@@ -341,6 +345,23 @@ try {
 
     try {
         if (isLinuxOrMac) {
+            if ($IsMacOS -and (-not $SkipVerify)) {
```

Comment:
```suggestion
            if ($IsMacOS -and !$SkipVerify) {
```

Reply:
FWIW, even the old way you don't need to put in parens; `-and -not $boolVar` would be the same but, yeah, I agree the `!` is concise.